linode/manager
# change: [UIE-8254] - Add tooltip for ipv6 for new db clusters

## Description 📝
Add a tooltip for Host on the Connection Details screen.

## Changes  🔄
- Added the tooltip.

## Target release date 🗓️
11/12/2024

## Preview 📷

| Before  | After   |
| ------- | ------- |
|  ![Screenshot 2024-11-07 at 2 59 08 PM](https://github.com/user-attachments/assets/4d070f7f-ad72-4b05-8ab7-951a9e54f7ca) | ![Screenshot 2024-11-07 at 2 58 10 PM](https://github.com/user-attachments/assets/8977a017-a2cf-419e-91e9-5a659242e8d4) |


## How to test 🧪
### Verification steps
(How to verify changes)
- Navigate to Database Landing page and view connection details

## As an Author I have considered 🤔

*Check all that apply*

- [ x] 👀 Doing a self review
- [ x] ❔ Our [contribution guidelines](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md)
- [ x] 🤏 Splitting feature into small PRs
